iree/samples/simple_embedding/BUILD

```diff
@@ -23,35 +23,15 @@ if(${IREE_TARGET_BACKEND_DYLIB-LLVM-AOT} AND ${IREE_HAL_DRIVER_DYLIB})
 
 # TODO(llvm-bazel/#228): re-enable bazel.
```

Comment:
Hm, this links to the internal issue tracker, correct? This is okay, as I don't care that much about bazel world, but it may be of value for other non-googlers if this is tracked as an GH issue.